server.js

```diff
@@ -23,64 +35,107 @@ app.use(function (req, res, next) {
 });
 
 function render(req, res) {
-	if (req.text === undefined) {
-		res.status(400).send('The body must have Content-Type:text/html');
-		return;
-	}
+	if (req.is('application/json')) {
```

Comment:
We don't want two big branches here. Rather, extract all the parameters at the start of the function, and then just have a single chunk of code that launches chrome and performs the rendering.